crates/matrix-sdk/tests/integration/encryption/recovery.rs

```diff
@@ -780,11 +780,15 @@ async fn test_reset_identity() {
     assert_eq!(client.encryption().recovery().state(), RecoveryState::Enabled);
 
     // Disabling backups
-    Mock::given(method("DELETE"))
-        .and(path("_matrix/client/r0/room_keys/version/1"))
+    Mock::given(method("GET"))
```

Comment:
You are not quite testing the happy path if you never need to mock the `DELETE` endpoint. You're returning a 404 on the GET version call, so the SDK concludes that a backup doesn't exist and doesn't need to be deleted.

A valid test, but if we had a test for the happy path, which does need to delete the backup as well.